x-pack/agent/pkg/agent/application/router.go

```diff
@@ -104,6 +104,8 @@ func (r *router) Dispatch(id string, grpProg map[routingKey][]program.Program) e
 			continue
 		}
 
+		r.log.Debuf("Removing routing key %s", k)
```

Comment:
![Image of Yaktocat](https://upload.wikimedia.org/wikipedia/commons/thumb/b/bc/Shia_LaBeouf_at_TIFF_2017_%2836949725591%29_%28cropped%29.jpg/440px-Shia_LaBeouf_at_TIFF_2017_%2836949725591%29_%28cropped%29.jpg)